src/components/Trip/EditTrip/EditTrip.jsx

```diff
@@ -193,11 +192,30 @@ const EditTrip = () => {
 		getTrip();
 	}, []);
 
+    const handleCancel = () => {
+		navigate('/');
```

Comment:
Ja bym wracała chyba navigate(-1) do pełnego opisu podróży, żeby użytkownik mógł od razu zobaczyć, jak wygląda to po edycji - ale to moja preferencja, więc też się nie upieram :)

Reply:
zmieniłam obecnie na navigate('/mytrips/ownedtrips'); - moze byc? :D

Reply:
Może po zapisaniu zamian `navigate(`/trip/${tripID}`);` ?

Reply:
navigate(-1) powinien dać ten sam efekt co navigate(/trip/${tripID})

Reply:
done